plugins/LilyPad/Tooltips.cpp

```diff
@@ -63,8 +63,13 @@ LPWSTR dialog_message(int ID, bool *updateText)
                    L"Double-click a device in the list or right-click it and select \"Test Device\" to display a continuously updated list of the state of all inputs on the selected device.\n"
                    L"Use this option to check if all the inputs on a controller function properly.\n\n"
                    L"Right-click and select \"Refresh\" to update the list of devices in case a recently connected device has not shown up yet.";
-        case IDC_G_XI:
-            return L"Xbox 360/Xbox One controllers (or devices/software imitating such) only";
+        case IDC_G_DI:
+			return L"(Legacy) Enable this if your gamepad doesn't support Xinput.\n\n"
+				   L"This will check your inputs (button presses) and maps them to corresponding action";
+		case IDC_G_XI:
+            return L"Imitates a Xbox (360) controller for a wide range of controllers.\n\n"
+				   L"The computer doesn't notice even if it's a totally different controller.\n\n"
+				   L"Steam also uses this method (but Steam also convert non-compatible controllers";
```

Comment:
Maybe
```
        case IDC_G_XI:
            return L"Xbox 360/Xbox One controllers (or devices/software imitating such) only.\n\n";
                       L"Running through Steam might be required for the controllers to work properly.";
```

Reply:
While that is kinda true, the first sentence feels off. Perhaps removing the word 'only' because it feels redundant?
The brackets help readability however the meaning is a bit lost to me. (or devices/software imitating such as other controllers)